desc/compute/utils.py

```diff
@@ -0,0 +1,306 @@
+import warnings
+from termcolor import colored
+
+from desc.backend import jnp
+from desc.grid import LinearGrid
+
+
+def _get_proper_surface(grid, surface_label):
+    """Returns grid quantities associated with the given surface label.
+
+    Parameters
+    ----------
+    grid : Grid
+        Collocation grid containing the nodes to evaluate at.
+    surface_label : str
+        The surface label of rho, theta, or zeta.
+
+    Returns
+    -------
+    :rtype: (ndarray, ndarray, ndarray)
```

Comment:
i dont think we use RST tags elsewhere in the code, they tend not to play nice with numpydoc

Reply:
Ok I removed them both places I used :rtype:.